docs/docs/50-user-guide/60-reference-docs/80-webhook-receivers/dockerhub/index.md

```diff
@@ -0,0 +1,103 @@
+---
+sidebar_label: Docker Hub
+description: How to configure Docker Hub webhooks with Kargo for automated artifact discovery.
+---
+
+# Docker Hub Webhook Receiver
+
+The Docker Hub Webhook Receiver responds to `push` events originating from Docker Hub repositories.
+
+In response to a `push` event, the receiver "refreshes" `Warehouse` resources subscribed to the corresponding Docker Hub repository.
+
+:::info
+"Refreshing" a `Warehouse` means enqueuing it for immediate reconciliation by the Kargo controller, which will attempt to discover new artifacts from all subscribed repositories.
+:::
+
+:::note
+Unlike GitHub, Docker Hub does **not** sign webhook payloads. Kargo uses a static token for basic validation.
+:::
+
+## Configuring the Receiver
+
+To enable webhook support for Docker Hub, you must configure a Kubernetes `Secret` and reference it in your `ProjectConfig`.
+
+The `Secret` must include a `dockerhub-secret` key under `stringData`. This value acts as a token for lightweight validation of incoming webhook requests.
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: dh-wh-secret
+  namespace: kargo-demo
+stringData:
+  dockerhub-secret: <your-static-token-here>
+````
+
+Next, reference this `Secret` in your `ProjectConfig`:
+
+```yaml
+apiVersion: kargo.akuity.io/v1alpha1
+kind: ProjectConfig
+metadata:
+  name: kargo-demo
+  namespace: kargo-demo
+spec:
+  webhookReceivers:
+    - name: dockerhub-receiver
+      dockerhub:
+        secretRef:
+          name: dh-wh-secret
+```
+
+## Retrieving the Receiver's URL
+
+After applying the `ProjectConfig`, Kargo will generate a unique, hard-to-guess URL for the Docker Hub webhook receiver.
+
+You can retrieve this URL using the following command:
+
+```shell
+kubectl get projectconfigs kargo-demo \
+  -n kargo-demo \
+  -o=jsonpath='{.status.webhookReceivers}'
+```
+
+Copy the URL associated with your Docker Hub webhook receiver. You'll use this in the next step.
```

Comment:
```suggestion
```

This is a reference document, and sections in this document are typically standalone. Because of this, instructions to "copy" something should not be made.

Instead, within "registering with Docker Hub", you would reference back to this section to instruct to user in how to obtain the value.